pkgs/development/libraries/libfm/default.nix

```diff
@@ -8,8 +8,9 @@
 , pkg-config
 , vala
 , extraOnly ? false
-, withGtk3 ? true , gtk3, gtk2
+, withGtk3 ? false, gtk2, gtk3
```

Comment:
You are sure about this change?

Reply:
It triggers other dependencies, and they call both libfm and gtk. If they aren't in sync, the build fails.

An example is lxpanel. It doesn't build with libfm-gtk3.

Reply:
(Maybe LXDE needs some "global" variable to handle this.)